src/Pim/Bundle/CatalogBundle/Resources/elasticsearch/index_configuration.yml

```diff
@@ -150,6 +150,12 @@ mappings:
                     match_mapping_type: 'boolean'
                     mapping:
                         type: 'boolean'
+            -
+                date:
+                    path_match: 'values.*-date.*'
+                    match_mapping_type: 'string'
+                    mapping:
+                        type: 'keyword'
```

Comment:
Because I didn’t know it exists :D. I was working on the completeness when you guys discussed these topics.